core/src/ldml/C7043_ldml.md

```diff
@@ -0,0 +1,165 @@
+# KMX Plus Binary Format #7043
```

Comment:
i'm fine to move this file.  Just very convenient to do all my work in the repo.

Reply:
Fine by me -- can you update the issue to point to the appropriate location where the most up-to-date version of the spec can be found?